packages/@ember/string/index.ts

```diff
@@ -111,6 +112,16 @@ function _fmt(str: string, formats: any[]) {
   @public
 */
 export function loc(str: string, formats: any[]): string {
+  deprecate('loc is deprecated.', false, {
```

Comment:
This is pretty short and sweet. Do you think we need to say something a bit longer here?

Reply:
I'm finishing the deprecation guide and then I'll update the ember.js side with the final link and text!